src/node/internal/crypto_random.ts

```diff
@@ -142,7 +142,7 @@ export function randomFill(buffer: ArrayBufferView|ArrayBuffer,
   // We're currently not actually implementing the fill itself asynchronously,
   // so we defer to randomFillSync here, but we invoke the callback asynchronously.
   new Promise<void>((res) => {
-    randomFillSync(buffer, offset, size);
+    randomFillSync((buffer as Uint8Array), offset, size);
```

Comment:
is this casting necessary?

Above you're updating the `randomFillSync` function's `buffer` parameter type to `NodeJS.ArrayBufferView` and that is the type of `buffer` isn't it?

Reply:
Yeah you're right. I missed this.